tutorials/preprocessing/plot_50_artifact_correction_ssp.py

```diff
@@ -228,8 +228,8 @@
 # projectors for magnetometers, gradiometers, and EEG channels (default is two
 # projectors for each channel type).
 # :func:`~mne.preprocessing.compute_proj_ecg` also returns an :term:`events`
-# array containing the sample numbers corresponding to the onset of each
-# detected heartbeat.
+# array containing the sample numbers corresponding to the peak of the R wave
```

Comment:
can we link "the R wave" to something here?  Maybe https://en.wikipedia.org/wiki/QRS_complex ?

Reply:
Great idea! Will do.